.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+Thanks for stopping by to let us know something could be better!
+
+**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.
+
+Please run down the following list and make sure you've tried the usual "quick fixes":
+
+  - Search the issues already opened: https://github.com/googleapis/google-cloud-python/issues
+  - Check for answers on StackOverflow: http://stackoverflow.com/questions/tagged/google-cloud-python
+
+If you are still having issues, please be sure to include as much information as possible:
+
+#### Environment details
+
+1. Specify the API at the beginning of the title (for example, "BigQuery: ...")
+   General, Core, and Other are also allowed as types
+2. OS type and version
+3. Python version and virtual environment information: `python --version`
+4. google-cloud-<service> version:  `pip show google-<service>` or `pip freeze`
+5. Stacktrace if available
```

Comment:
Can we move this to its own section, below the code example?  That makes the flow more natural.

Reply:
done!